llvm/lib/Target/AMDGPU/SIWholeQuadMode.cpp

```diff
@@ -1582,6 +1583,28 @@ void SIWholeQuadMode::lowerInitExec(MachineInstr &MI) {
   MachineBasicBlock *MBB = MI.getParent();
   bool IsWave32 = ST->isWave32();
 
+  if (MI.getOpcode() == AMDGPU::SI_INIT_WHOLE_WAVE) {
+    // TODO: Assert that it's in the entry block
```

Comment:
Do the TODO? :)

Reply:
Ooops, I thought for sure I would notice that before pushing...